### PR TITLE
Stackable tabs

### PR DESCRIPTION
**Pull request by Perkovec**
ide

### CODE REVIEW

**Comment by Perkovec** [comment]
- [ ] Scrolling
- [ ] Stacked tabs after active tab

**Comment by Perkovec** [comment]
ide

**Comment by Perkovec** [comment]
Sorry, but i don't have idea how make scrolling of stacked tabs 😓

**Comment by Perkovec** [comment]
@egoist completed

**Comment by egoist** [comment]
It should resize after I closed some tabs, or bad thing happens..

**Comment by Perkovec** [comment]
@egoist fixed

File: src/components/header.vue [hunk 1]
```diff
@@ -221,12 +244,23 @@
...
     },
+    ready() {
+      this.listenEvents()
+    },
+    computed: {
+      header() {
+        return $('.header')
+      },
+      tabContainer() {
+        return $('.tab-container')
```
Comment (egoist) [inline src/components/header.vue]: why computed property?

Comment (Perkovec) [inline src/components/header.vue]: @egoist use `data`?

Comment (egoist) [inline src/components/header.vue]: why not simply reference `$('.tab-container')` when you use it?

Comment (Perkovec) [inline src/components/header.vue]: for optimization :)

Comment (egoist) [inline src/components/header.vue]: what about using `$els` ?

Comment (Perkovec) [inline src/components/header.vue]: @egoist fixed

**Comment by egoist** [comment]
still there

gif

**Comment by Perkovec** [comment]
@egoist what actions here?

**Comment by egoist** [comment]
<kbd>cmd + w</kbd>

**Comment by Perkovec** [comment]
@egoist fixed

**Comment by egoist** [comment]
fmkp70c7IIb9C

**Pull request merged**